app/utils/InstantProvider.scala

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import java.time.Instant
```

Comment:
Use 'java.time.Clock' instead of this?

https://stackoverflow.com/questions/42876651/scala-dependency-injection-in-play-framework